lib/rmg/QuestArtifactPlacer.cpp

```diff
@@ -38,8 +38,7 @@ void QuestArtifactPlacer::addQuestArtZone(std::shared_ptr<Zone> otherZone)
 
 void QuestArtifactPlacer::addQuestArtifact(const ArtifactID& id)
 {
-	logGlobal->info((boost::format("Need to place quest artifact artifact %s")
-		% VLC->arth->getById(id)->getNameTranslated()).str());
+	logGlobal->info("Need to place quest artifact artifact %s", VLC->arth->getById(id)->getNameTranslated());
```

Comment:
@DjWarmonger why not VLC->artifacts() instead of VLC->arth? I think using entities is better when we can.